stix_shifter_modules/splunk/stix_translation/transformers.py

```diff
@@ -46,3 +46,20 @@ def get_pair_of_hash(hash_raw):
                 return obj
             hashes = dict(map(lambda x: get_pair_of_hash(x), hashes))
             return hashes
+
+
+class SplunkMacFormatChange(ValueTransformer):
+    """    A value transformer for converting MAC value into stix format(using : separator)  """
+
+    @staticmethod
+    def transform(macvalue):
+        """correcting mac address presentation, it should be 6 octate separated
+         by only colon (:) not by any other special character """
+        macvalue = re.sub("[^A-Za-z0-9]", "", macvalue)
```

Comment:
Isn't mac address allowed literals [A-Fa-f0-9] instead of A-Z and a-z?

Reply:
As suggested mac address allowed literals changed to [A-Fa-f0-9].